pkg/provider/resource_fareplica.go

```diff
@@ -0,0 +1,369 @@
+package provider
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/EnterpriseDB/terraform-provider-biganimal/pkg/api"
+	"github.com/EnterpriseDB/terraform-provider-biganimal/pkg/models"
+	"github.com/EnterpriseDB/terraform-provider-biganimal/pkg/utils"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+type FAReplicaResource struct{}
+
+func NewFAReplicaResource() *FAReplicaResource {
+	return &FAReplicaResource{}
+}
+
+func (c *FAReplicaResource) Schema() *schema.Resource {
+	return &schema.Resource{
+		//todo: Description: "The cluster resource is used to manage BigAnimal clusters. See [Creating a cluster](https://www.enterprisedb.com/docs/biganimal/latest/getting_started/creating_a_cluster/) for more details.",
```

Comment:
You would probably refer to [Managing Replicas EDB Doc](https://www.enterprisedb.com/docs/biganimal/latest/using_cluster/managing_replicas/) in the description.

Reply:
Thanks, done.